pennylane/_qubit_device.py

```diff
@@ -854,6 +854,13 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
             if not hasattr(m.obs, "base_name"):
                 m.obs.base_name = None  # This is needed for when the observable is a tensor product
 
+        if self.shots is not None:
+            warnings.warn(
+                "Adjoint diff called on device with finite shots. Adjoint diff always"
```

Comment:
Would be great to rephrase the message a bit such that the term "adjoint differentiation" aligns well in there (e.g., "Requested the gradient to be computed with finite shots, this computation is always exact when using the adjoint differentiation method." or so).

Reply:
I'll think about this a bit more.

Reply:
Or I'll just use what you said ;)